src/request/providers/xhr.ts

```diff
@@ -18,6 +18,7 @@ import SubscriptionPool from '../SubscriptionPool';
  */
 export interface XhrRequestOptions extends RequestOptions {
 	blockMainThread?: boolean;
+	includeRequestedWithHeader?: boolean;
```

Comment:
Yeah, I was going to say README, but realise for core that is a ginormous task and actually for lots of it, just sane API docs is likely to be better, maybe...  Adding this particular flag to the README would be a bit crazy as it stands at the moment, since this is our `request` documentation at the moment:

> ### HTTP requests
>
> The `@dojo/core/request` module contains methods to simplify making http requests. It can handle making requests in both node and the browser through the same methods.